config/defaults.conf

```diff
@@ -366,100 +366,45 @@ use_classifier = ""  // Used to make some task (usually a probing task with no t
 glue-diagnostic { use_classifier = "mnli" }
 superglue-diagnostic { use_classifier = "rte-superglue" }
 
-rte = {}
-rte_classifier_hid_dim = 128
-rte_d_proj = 128
-rte_classifier_dropout = 0.4
-rte_pair_attn = 0
-rte_val_interval = 100
-rte_lr = 0.0003
-
-wnli = {}
-wnli_classifier_hid_dim = 128
-wnli_d_proj = 128
-wnli_classifier_dropout = 0.4
-wnli_pair_attn = 0
-wnli_val_interval = 100
-wnli_lr = 0.0003
-
-mrpc = {}
-mrpc_classifier_hid_dim = 256
-mrpc_d_proj = 256
-mrpc_classifier_dropout = 0.2
-mrpc_pair_attn = 0
-mrpc_val_interval = 100
-mrpc_lr = 0.0003
-
-sst = {}
-sst_classifier_hid_dim = 256
-sst_d_proj = 256
-sst_classifier_dropout = 0.2
-sst_val_interval = 100
-sst_lr = 0.0003
-
-cola = {}
-cola_classifier_hid_dim = 256
-cola_d_proj = 256
-cola_classifier_dropout = 0.2
-cola_val_interval = 100
-cola_lr = 0.0003
-
-sts-b = {}
-sts-b_classifier_hid_dim = 512
-sts-b_classifier_dropout = 0.2
-sts-b_pair_attn = 1
-sts-b_val_interval = 1000
-sts-b_lr = 0.0003
-
-sts-b-alt = {}
-sts-b-alt_classifier_hid_dim = 512
-sts-b-alt_classifier_dropout = 0.2
-sts-b-alt_pair_attn = 1
-sts-b-alt_val_interval = 1000
-sts-b-alt_lr = 0.0003
-
-qnli = {}
-qnli_classifier_hid_dim = 512
-qnli_classifier_dropout = 0.2
-qnli_pair_attn = 1
-qnli_val_interval = 1000
-qnli_lr = 0.0003
-
-qnli-alt = {}
-qnli-alt_classifier_hid_dim = 512
-qnli-alt_classifier_dropout = 0.2
-qnli-alt_pair_attn = 1
-qnli-alt_val_interval = 1000
-qnli-alt_lr = 0.0003
-
-mnli = {}
-mnli_classifier_hid_dim = 512
-mnli_classifier_dropout = 0.2
-mnli_pair_attn = 1
-mnli_val_interval = 1000
-mnli_lr = 0.0003
-
-mnli-alt = {}
-mnli-alt_classifier_hid_dim = 512
-mnli-alt_classifier_dropout = 0.2
-mnli-alt_pair_attn = 1
-mnli-alt_val_interval = 1000
-mnli-alt_lr = 0.0003
-
-qqp = {}
-qqp_classifier_hid_dim = 512
-qqp_classifier_dropout = 0.2
-qqp_pair_attn = 1
-qqp_val_interval = 1000
-qqp_lr = 0.0003
-
-qqp-alt = {}
-qqp-alt_classifier_hid_dim = 512
-qqp-alt_classifier_dropout = 0.2
-qqp-alt_pair_attn = 1
-qqp-alt_val_interval = 1000
-qqp-alt_lr = 0.0003
+tmpl-1 {
```

Comment:
Mind renaming these to something descriptive? Even if the pattern isn't totally clear, something like 'glue-small-tasks-tmpl-1' would help.